oecophylla/taxonomy/tests/data/shogun/combined.phylum.tsv

```diff
@@ -0,0 +1,28 @@
+	sampleA	sampleB
+k__Archaea;p__Crenarchaeota	1.0	0.0
```

Comment:
It might be the wrong context here, but I think we agreed on reporting NCBI taxonomy IDs as feature names for taxons. Because we can than compare the alternative profilers + assign taxonomy in a second step which would allow to collapse read counts on higher levels.

Reply:
I agree that this is a good idea (for now 😉). I think for SHOGUN, Ben said they have a mapping file linking their taxonomy strings to RefSeq IDs to NCBI TaxIDs?

Reply:
@tanaes They sent us something, including a lineaege-to-nucl map and a nucl-to-TaxID map. However they don't necessarily result in a lineage-to-TaxID map. We need TaxIDs for all ranks, not just tips (genomes). They also sent us some binaries for converting taxdump into lineage. Let me and @sjanssen2 play with them to see if we can get what we need.

Reply:
Note also the NCBITax functionality in ete3... it's pretty handy for these situations sometimes.

Reply:
Indeed looks quite handy! http://etetoolkit.org/docs/latest/tutorial/tutorial_ncbitaxonomy.html

By the way, wanna merge? @tanaes